releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -4,6 +4,99 @@
+
+```json
+{ 
+    "Name": "Big", 
+    "ConfigurationReference": "ShoppingCart:Big" 
+},  
+{ 
+    "Name": "Small", 
+    "ConfigurationValue": {
+        "Size": 300
+    }
+}
+```
+
+Variants are then assigned via Allocation, which is defined as a part of the feature flag. The fields are User, Group, Percentile, and a couple defaults.
+
+```json
+"Allocation": { 
+    "DefaultWhenEnabled": "Small", 
+    "DefaultWhenDisabled": "Small",  
+    "User": [ 
+        { 
+            "Variant": "Big", 
+            "Users": [ 
+                "Marsha" 
+            ] 
+        } 
+    ], 
+    "Group": [],
+    "Percentile": []
+}
+```
+
+A feature with Variants will still have a boolean result when called via `IsEnabledAsync`, following the usual filters. However, when a feature is called with `GetVariantAsync`, it will evaluate the allocation and return a `Variant` object. 
+
+```csharp
+Variant variant = await featureManager.GetVariantAsync(MyFeatureFlags.FeatureU);
+
+String variation = variant.Name;
+IConfigurationSection variantConfiguration = variant.Configuration;
+```
+
+For more details on Variants, see [here](https://github.com/microsoft/FeatureManagement-Dotnet/tree/release/v4?tab=readme-ov-file#variants).
+
+### Telemetry
+
+Telemetry offers the ability to emit events containing information about a feature evaluation. This can be used to ensure a flag is running as expected, or to see which users were given which features and why they were given the feature. To enable this functionality, two things need to be done.
+
+The flag needs to explicitly enable telemetry in its definition.
+
+```json
+"MyFlag": {
+    "Telemetry": {
+        "Enabled": true
+    }
+}
+```
+
+And a telemetry publisher needs to be registered. One is offered out-of-the-box for Application Insights and could be added with a single line.
+
+```csharp
+builder.services
+    .AddFeatureManagement()
+    .AddTelemetryPublisher<ApplicationInsightsTelemetryPublisher>();
+```
+
+An example is available to demonstrate how to use the new Telemetry in an ASP.NET application. See [the example](https://github.com/microsoft/FeatureManagement-Dotnet/tree/release/v4/examples/EvaluationDataToApplicationInsights) in the examples folder.
+
+For more details on Telemetry, see [here](https://github.com/microsoft/FeatureManagement-Dotnet/tree/release/v4?tab=readme-ov-file#telemetry).
+
+### Additional Changes
+
+#### IVariantFeatureManager
+
+`IVariantFeatureManager` has been added as the successor of the existing `IFeatureManager`. It continues to offer the functions of `IFeatureManager`, but offers the new `GetVariantAsync` methods as well.
+
+#### Cancellation Tokens
+
+`IVariantFeatureManager` incorporates cancellation tokens into the methods of `IFeatureManager`. To migrate from the old interface, simply use the new one, and adjust calls to `IsEnabledAsync` or `GetFeatureNamesAsync` to include a `CancellationToken`. 
```

Comment:
I wouldn't dub either interface old or new, but rather say to take advantage, use `IVariantFeatureManager` instead of IFeatureManager. If we were to own up to 'old' moniker we should deprecate.